changelog/2058.feature.rst

```diff
@@ -1 +1 @@
-Added a 'notches' keyword option to thomson.spectral_density, which allows users to notch out the output spectrum over a wavelength range. Also made some changes to the thomson fitting models to include the notch as an option.
+Added a 'notches' keyword option to ~plasmapy.diagnostics.thomson.spectral_density, which allows users to notch out the output spectrum over one or multiple wavelength ranges. Changed the ~plasmapy.diagnostics.thomson.spetctral_density_model function to allow notches to be applied during fitting.
```

Comment:
```suggestion
Added a ``notches`` keyword option to `~plasmapy.diagnostics.thomson.spectral_density`, which allows users to notch out the output spectrum over one or multiple wavelength ranges. Changed the `~plasmapy.diagnostics.thomson.spectral_density_model` function to allow notches to be applied during fitting.
```
The single back ticks will make it show up in the online docs with a link to the function, and the double back ticks will make it appear as an in-line code snippet.